### ISSUE TITLE
Why use NLT at all?

### ISSUE THREAD

**Issue by VisionaryMind** [google/neural-light-transport#9]
This is a very well-designed experiment, so please do not take my words to be overly critical. It is just that what you are proposing in your paper is not entirely true. You say:

"With this learned LT, one can relight the scene photorealistically with a directional light or an HDRI map, synthesize novel views with view-dependent effects, or do both simultaneously, all in a unified framework **using a set of sparse, previously seen observations**. "

This last statement would indicate that you could take a _small number_ of photographs from different perspectives (such as those presented in the instant-ngp model) and then relight a static observation / scene (i.e. single photograph). Reading more deeply into your paper, this proposal is contradicted. In fact, not only does this project require OLAT data as an input (specifically, _hundreds_ of OLAT images from over 60 RGB cameras placed around the performer) --- which could hardly be qualified as "sparse" -- but it appears you also require a proxy mesh (distributed in a Blender file). I do not see that any rendering is possible without a mesh --- and a reliable, high-resolution mesh cannot be generated _using a set of sparse, previously seen observations_, as you state. 

To make a long story short, all of this could already be done many years ago using photogrammetry and GPU-based rendering engines (such as Cycles, Redshift, and Octane). Perhaps you can explain to me why we would want to take the time to invest thousands of dollars in an OLAT-capture cage when in just a fraction of the time with a single DSLR camera (and a bit of patience), the same quality output could be achieved? This isn't theoretical, I have done it.

In other words, this project would only be useful if it were able to relight scenes from a small set of images. Clearly, from the questions being posted here, the average reader has misunderstood what the original paper was proposing. Perhaps if the language were simplified or at least brought closer to reality, we wouldn't waste our time looking at the code thinking that it could be used to "relight a scene from a set of sparse views", OLAT-derived or not. _It cannot._ 

TL;DR -- instead of downloading 19GB of "rendered data" for a dragon, just take a 2Mb OBJ of the same dragon, import it into Blender, put some point-lights / HDRI around it, and render it in Cycles. No need to mess with "neural light transport", which would require 1) having access to a dragon figurine and 2) taking hundreds of photos of that dragon with actual lights (in a studio or OLAT cage), crossing your fingers, and hoping for the same results as the OBJ in Cycles.

It is likely I am missing the point of this paper. It seems silly to go to so much trouble to do something that has another more simplistic approach that has been around for over a decade now.

**Comment by xiumingzhang** [comment]
Thanks for your comments.

We made it clear that the model is per person, meaning that you won't be able to relight person B by training on person A. Many other projects aim to do "one/zero-shot relighting," as you called it.

For a synthetic scene like the dragon Blender file, sure, you can render whatever you want with the 2 MB .obj file, similar to how you can render AAA game-level synthetic scenes from any viewpoint under any illumination, but that's not the point of the paper. The point here is to interpolate lights and views for a person (whose reflectance is hard to specify by hand, as opposed to the Blender dragon that has just a simple BRDF) when given an OLAT capture that includes an _imperfect_ shape proxy (as opposed to the perfect dragon geometry). We couldn't release our human dataset due to privacy issues; the dragon dataset is just an example people can follow if they want to adapt our code for their own data. To summarize, if you have a synthetic scene over which you have complete control in every aspect (shape, reflectance, illumination), then you are right about just rendering it with Blender. But when you have only a shape _proxy_ (with artifacts from scanning) and some simplified reflectance model, and you throw them into Blender, you will end up with artifacts due to imperfect geometry (e.g., bumps) and reflectance (e.g., missing SSS unless you specify that manually to your reflectance model).

A concrete example of what I said above: Suppose you want to relight a specific static actor from any viewpoint. Given your experience in photogrammetry and rendering, you can "scan" the shape of this actor using MVS or similar techniques and also acquire some PBR maps by assuming certain reflectance properties. Because your "scanned" geometry is imperfect, and your reflectance models are simplified (up to what PBR maps you acquired and what effects these maps can represent), if you import this model into Blender and just render that, then you will end up with artifacts due to the imperfect shape and reflectance. On the contrary, NLT doesn't explicitly model BRDFs, is tolerant of geometric errors, and therefore may avoid the said artifacts.
